test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -422,6 +422,18 @@ public void It_migrates_project_with_output_name()
                 .Count().Should().Be(1);
         }
 
+        [Fact]
+        public void It_migrates_and_builds_library_without_netstandard_reference()
+        {
+            string projectName = "LibraryWithoutNetStandardLibRef";
+            var projectDirectory = TestAssetsManager.CreateTestInstance(projectName, callingMethod: $"It_migrates_{projectName}").Path;
```

Comment:
Why do we need to override the calling method? It's easier to see what's going on if it's just... the calling method